### PR TITLE
typo db connection

### PR DESCRIPTION
**Pull request by doorooful**
- db connection defined on module adapter('module_lab')
- typo module code has been refactored regarding db schema

### CODE REVIEW

**Comment by doorooful** [comment]
### Database screenshot of typo module result
Screenshot 2023-10-14 at 3 50 10 AM

File: dt-demo-be/document.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def run(self):
                             + f"{self.__active_modules} starting ...")
         args = modules.argparse.Namespace(
             modules=self.__active_modules,
-            sentences=[s for s in self.__sentences]
+            sentences=[s for s in self.__sentences],
```
Comment (yoouung) [inline dt-demo-be/document.py:115]: If `,` is not essential, wouldn't it be better to leave it out?

Comment (doorooful) [reply dt-demo-be/document.py]: Applied :)

**Comment by doorooful** [comment]
Please find the recent changes 😃

**Review by yoouung — APPROVED** [review]
👍🏻 👍🏻

**Pull request merged**